oda_api/api.py

```diff
@@ -1339,3 +1340,48 @@ def from_response_json(cls, res_json, instrument, product):
                 p.meta_data = p.meta
 
         return d
+
+class ProgressReporter(object):
+    """
+    The class allows to report task progress to end user
+    """
+    def __init__(self):
+        self._callback = None
+        callback_file = ".oda_api_callback"  # perhaps it would be better to define this constant in a common lib
```

Comment:
To be more general, it could be the init parameter with default value. Also, probably good to have another parameter to be able to set the url directly. (Even though the current usage is by passing the file of fixed name)

Reply:
This is all somehow all outside intended usage. Interface with nb2workflow should be quite fixed. In can be changed in the future if needed, but there is no use case at the moment, is it, @dsavchenko ?

Reply:
I agree that there is no use case at the moment. It was just a proposal

Reply:
We keep it in mind!